File: CS/DXApplication1/Program.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using DevExpress.UserSkins;
using DevExpress.Skins;
using DevExpress.LookAndFeel;
using DXApplication.Data;

namespace DXApplication {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main() {
            ConnectionSettings.SetUp("your_server_name.database.windows.net", "your_login", "your_password", "your_database_name");
            ConnectionSettings.SelectQuery = "SELECT * FROM SalesLT.Product";
            EFDataProvider.PreInitContext();

            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Main());
        }
    }
}

File: VB/DXApplication1/Program.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.UserSkins
Imports DevExpress.Skins
Imports DevExpress.LookAndFeel
Imports DXApplication.Data

Namespace DXApplication
	Friend Module Program
        ''' <summary>
        ''' The main entry point for the application.
        ''' </summary>
        <STAThread>
        Sub Main()
            ConnectionSettings.SetUp("your_server_name.database.windows.net", "your_login", "your_password", "your_database_name")
            ConnectionSettings.SelectQuery = "SELECT * FROM SalesLT.Product"
            EFDataProvider.PreInitContext()

            Application.EnableVisualStyles()
            Application.SetCompatibleTextRenderingDefault(False)
            Application.Run(New Main())
        End Sub
    End Module
End Namespace
